packages/babeltrace2/package.py

```diff
@@ -0,0 +1,75 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from os.path import join as pjoin
```

Comment:
You can remove this line. In SPRs you should be using `join_path()`. https://spack.readthedocs.io/en/latest/package_api.html#spack.package.join_path